src/services/TalkCommentEmailService.php

```diff
@@ -18,10 +18,25 @@ public function __construct($config, $recipients, $talk, $comment)
     public function sendEmail()
     {
         $this->setSubject("New feedback on " . $this->talk['talk_title']);
+
+        $byLine = '';
+
+        // NOTE: All comments sent on web2 now seem to require login, is this
+        // condition required?
+        if(isset($this->comment['user_display_name'])) {
+            $byLine = ' by ' . $this->comment['user_display_name'];
+
+            if(isset($this->comment['username'])) {
+                $byLine .= ' (' . $this->comment['username'] . ')';
+            }
+        }
+
```

Comment:
Please default `$byLine` to either blank or 'by anonymous' so that we don't don't come a cropper if we ever enable anonymous commenting